.travis.yml

```diff
@@ -1,18 +1,18 @@
-language: java
-
 sudo: required
 
 matrix:
   include:
-    - language: java
-      install: true
+    - language: node_js
```

Comment:
This won't work. With language as 'node_js' but no node version, you'll get 0.10, which Appium will not install with. Further, `jdk` will be ignored.

Reply:
Yes